Artifacts/PgSql/Data/Security/2110-DataStandard4.0-ResourceClaimMetadata.sql

```diff
@@ -40,7 +40,7 @@ BEGIN
         RAISE NOTICE 'Creating new claim:studentAssessmentEducationOrganizationAssociation';
 
         INSERT INTO dbo.ResourceClaims(DisplayName, ResourceName, ClaimName, ParentResourceClaimId, Application_ApplicationId)
-        VALUES ('educationOrganizationAssociationTypeDescriptor', 'educationOrganizationAssociationTypeDescriptor', 'http://ed-fi.org/ods/identity/claims/educationOrganizationAssociationTypeDescriptor', relationshipBasedDataResourceClaim_Id, application_Id);
+        VALUES ('studentAssessmentEducationOrganizationAssociation', 'studentAssessmentEducationOrganizationAssociation', 'http://ed-fi.org/ods/identity/claims/studentAssessmentEducationOrganizationAssociation', relationshipBasedDataResourceClaim_Id, application_Id);
```

Comment:
This script started to fail after splitting the PgSql ResourceClaimMetadata_generated script